tls/test/test_hello_message.py

```diff
@@ -236,14 +256,11 @@ def test_parse_server_hello(self):
 
     def test_parse_server_hello_extensions(self):
         """
-        :func:`parse_server_hello` returns an instance of :class:`ServerHello`
-        with extensions, when the extension bytes are present in the input.
+        :func:`parse_server_hello` fails to parse when
+        SIGNATURE_ALGORITHMS extention bytes are present in the packet
```

Comment:
Looks like the docs build failure is due to `extention`:

https://travis-ci.org/pyca/tls/jobs/176794154#L433